migrations/standardize-labels.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Sweet, I'm testing this now (please add `+x`):

```
❯ RELEASE_NAME=stackstorm-ha NAMESPACE=stackstorm migrations/standardize-labels.sh
zsh: permission denied: migrations/standardize-labels.sh
```

Reply:
Doh. Good catch!